lib/modules/commentTools.js

```diff
@@ -832,6 +832,17 @@ const magicPlaceholders = [
 			return date.toDateString();
 		},
 	}, {
+		matches: ['flair'],
+		description: 'On comments pages, returns the flair text of the original link',
+		handle() {
+			if (isPageType('comments')) { 
+				var linkflair = document.querySelector('.linkflairlabel').innerHTML;
```

Comment:
For bonus points, you could create a new Thing from the OP and use the Thing's method to get the link flair text.